### DISCUSSION TITLE
Support for Gujarati language

### DISCUSSION THREAD

**Discussion by himeshp** [deepgram/community#350] [Feature Requests] [answered]
Are there any plans to add support for Gujarati - Local Indian language?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @himeshp, while Deepgram's proprietary models don't support Gujarati, our managed Whisper model does. You can submit API requests with the URL parameters `model=whisper&language=gu`. See more details on our [DG Whisper Cloud Quickstart](https://developers.deepgram.com/docs/deepgram-whisper-cloud). Note that while Whisper accepts Gujarati audio, it's not an officially supported language, so you'll have to see whether the accuracy meets your needs.
